index.js

```diff
@@ -82,6 +84,14 @@ class ServerlessCustomDomain {
     this.targetHostedZoneName = this.givenDomainName.substring(this.givenDomainName.indexOf('.') + 1);
   }
 
+  setEndpointType(endpointType) {
+    this.endpointType = endpointType || 'EDGE';
+  }
+
+  setAcmRegion(region) {
+    this.acmRegion = region || 'us-east-1';
```

Comment:
Should this check that the `endpointType` is `REGIONAL`?

Reply:
I though about this but didn't come to a conclusion yet, if only endpointType REGIONAL is provided without acm region, or some other region than us-east-1 we can throw an exception. 

What I have a problem with is if endpointType EDGE is provided with some acm region. We can choose to just ignore it, but that doesn't quite feel right because we now have a parameter that has no impact whatsoever on the domain and maybe the user isn't aware of it. maybe we could add a warning in this case letting the user know...

For now, I covered it with a short explanation in the documentation regarding the relationship between the endpoint type and the acm region.

wdyt